data/json/items/comestibles/junkfood.json

```
@@ -462,6 +462,16 @@
     "charges": 5,
     "fun": 6
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "candy_bracelet",
+    "name": { "str_sp": "candy bracelet" },
+    "looks_like": "copper_bracelet",
+    "color": "yellow",
+    "description": "A candy bracelet made by stringing candies onto a thread. In theory, you could both wear and eat the bracelet.",
```

```suggestion
    "description": "A candy bracelet made by stringing candies onto a thread.  In theory, you could both wear and eat the bracelet.",
```
Double space after a full stop

---

done, i appreciate the advice